viewer/packages/cad-styling/src/prioritized/EmptyAreaCollection.test.ts

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import { EmptyAreaCollection } from './EmptyAreaCollection';
+
+import * as THREE from 'three';
+
+describe('EmptyAreaCollection', () => {
+  test('empty area collection does empty area collection stuff', () => {
+    const areaCollection = EmptyAreaCollection.instance();
+
+    expect(areaCollection).toBeTruthy();
+
+    expect(() => areaCollection.addAreas([new THREE.Box3()])).toThrowError();
+    expect(() => areaCollection.intersectWith([new THREE.Box3()])).toThrow();
```

Comment:
Should this throw? isn't intersection with an empty area collection just always an empty area collection?